.github/workflows/error-ref-publisher.yaml

```
@@ -0,0 +1,63 @@
+name: Error Code Reference Publisher
+on:
+  push:
+    branches:
+      - "master"
+    paths:
+      - '**.go'
+
+jobs:
+  error-ref:
+    name: Error codes utility
+    if: github.repository == 'meshery/meshery-nsm'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # token here with write access to meshkit repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+         go run github.com/meshery/meshery-nsm/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```

```suggestion
         go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```
No need to change this. This is running the error codes utility present in meshkit repo.